src/webgl/p5.RendererGL.js

```diff
@@ -1097,65 +1044,97 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
       this.filterShader = args[0];
     }
 
-    pg.clear(); // prevent undesirable feedback effects accumulating secretly
+    // Setting the target to the framebuffer when applying a filter to a framebuffer.
+
+    const target = this.activeFramebuffer() || this;
+
+    // Resize the framebuffer 'pg' and adjust its pixel density if it doesn't match the target.
+
+    if (
+      pg.width !== this.width ||
+      pg.height !== this.height
+    ) {
+      // Resize pg
+      pg.resizeCanvas(this.width, this.height);
+    }
+    if (
+      pg.pixelDensity() !== this._pInst.pixelDensity()
+    ) {
+      pg.pixelDensity(this._pInst.pixelDensity());
+    }
+
+    // Set the yScale of the filterCamera for framebuffers.
+    if (target !== this) {
+      this.filterCamera.yScale = this._curCamera.yScale;
```

Comment:
I think we're always using the filterCamera on a framebuffer now so I think its yScale should always be -1 (which is what your logic currently does as well.) But maybe a simpler way of achieving the same thing would be, when we define `this.filterCamera`, to say `this.filterCamera = this.getFilterGraphicsLayer().createCamera()` -- that way, its size will always match the size of the filter graphic when it resizes, and we won't need to touch its `yScale` value since it will be initialized to the correct value.